internal/module/helper.go

```diff
@@ -37,14 +30,14 @@ func IntermediateImageName(name, namespace, targetImage string) string {
 
 // ShouldBeBuilt indicates whether the specified KernelMapping of the
 // Module should be built or not.
-func ShouldBeBuilt(modSpec kmmv1beta1.ModuleSpec, km kmmv1beta1.KernelMapping) bool {
-	return modSpec.ModuleLoader.Container.Build != nil || km.Build != nil
+func ShouldBeBuilt(km kmmv1beta1.KernelMapping) bool {
```

Comment:
If we agree that a Module's build section should be used as default when it's not defined in the kernel mapping, then should this function accept a [*Build](https://github.com/kubernetes-sigs/kernel-module-management/blob/main/api/v1beta1/module_types.go#L47) instead?

Reply:
same answer as above